portal-web/test/functional/com/liferay/portalweb/macros/ContentPerformance.macro

```diff
@@ -3,5 +3,18 @@ definition {
 	macro openContentPerformance {
 		Click(locator1 = "ContentPerformance#CONTENT_PERFORMANCE_BUTTON");
 	}
+	
+	macro contentPerformanceMessage {
+		AssertTextEquals(
+			key_message = "${message}",
+			locator1 = "ContentPerformance#CONTENT_PERFORMANCE_MESSAGE",
+			value1 = "${message}");
+	}
 
+	macro contentPerformanceWithoutMessage {
+		AssertElementNotPresent(
+			key_message = "${message}",
+			locator1 = "ContentPerformance#CONTENT_PERFORMANCE_MESSAGE",
+			value1 = "${message}");
+	}
```

Comment:
There's no need to create a macro to assert a message, you can call the Assert method in the .testcase file.